CMakeLists.txt

```diff
@@ -74,6 +75,13 @@ else()
     -Wuninitialized
     -Wvla
   )
+  if(EXPORT_DYNAMIC)
+    if(CMAKE_C_COMPILER_ID MATCHES "Clang")
+      add_link_options("-Wl,-export-dynamic")
```

Comment:
better to use `-rdynamic` on linux and bsd systems it got lost on Makefile migration. 
I think macOS needs `-dynamic`, I can prepare a PR to restore the old behavior.

Reply:
Sounds good, thanks for the review. I am currently trying to figure out which compiler/OS combination needs which options - hence it's a draft PR for now.

Reply:
This should already enable backtrace dynamic linking, its currently disabled for release builds:

https://github.com/baresip/re/blob/main/cmake/re-config.cmake#L175